### ISSUE TITLE
10.0.22621.3374 not supported

### ISSUE THREAD

**Issue by billchenbest** [stascorp/rdpwrap#2528]
Windows 11 10.0.22621.3374 is not supported.

**Comment by TanMan1217** [comment]
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/14766828/termsrv.zip)

**Comment by manuel231509** [comment]
Where is the .dll pasted?

**Comment by TanMan1217** [comment]
> Where is the .dll pasted?

Did you not see my post above yours? That's the termsrv.dll from 10.0.22621.3374

**Comment by manuel231509** [comment]
If I saw it, it means that "10.0.22621.3374" also appears unsupported for me,
I saw the post, but I don't know exactly what to do with the .dll.

**Comment by manuel231509** [comment]
or what is the path of the termsrv.dll, in which the .dll is hosted

**Comment by TanMan1217** [comment]
I understand now. The termsrv.dll is in C:\Windows\System32. But I already posted it.

**Comment by manuel231509** [comment]
ok thanks, now I am trying to paste the .dll in the mentioned path and it does not allow replacing the .dll

**Comment by manuel231509** [comment]
What will it be due to?

**Comment by TanMan1217** [comment]
You do NOT paste this DLL. This DLL is used to figure out what values to add to the RDPWRAP.ini file. Once that new RDPWRAP.ini file is posted, you download that and replace the RDPWRAP.ini file in the RDP WRAPPER program folder.

**Comment by 77348953** [comment]
Does anyone have the RDPWRAP.ini ?

**Comment by Mushino0** [comment]
looking also for .ini file :)

**Comment by lucky1luc** [comment]
also looking for ini parameters

**Comment by sashaqwert** [comment]
Try https://github.com/llccd/RDPWrapOffsetFinder/releases

Execute EXE and add values to your `rdpwrap.ini` :)

Video: https://t.me/rdpwrap/69189

**Comment by jonathan19091** [comment]
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/14769848/termsrv.zip)

**Comment by dyachek** [comment]
[10.0.22621.3374]
SingleUserPatch.x64=1
SingleUserOffset.x64=C3F3
SingleUserCode.x64=Zero
DefPolicyPatch.x64=1
DefPolicyOffset.x64=1C2E5
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=9D661
LocalOnlyCode.x64=jmpshort
SLInitHook.x64=1
SLInitOffset.x64=29ED0
SLInitFunc.x64=New_CSLQuery_Initialize

[10.0.22621.3374-SLInit]
bServerSku.x64=12BF50
bRemoteConnAllowed.x64=12BF64
bFUSEnabled.x64=12BF74
bAppServerAllowed.x64=12BF5C
bMultimonAllowed.x64=12BF68
lMaxUserSessions.x64=12BF54
ulMaxDebugSessions.x64=12BF70
bInitialized.x64=12BF4C

**Comment by lucky1luc** [comment]
@dyachek Neither of these entries is for version 10.0.22621.3374 of termsrv.dll

**Comment by Sharkta2** [comment]
I'm also looking for an update to rdpwrap.ini for 10.0.22621.3374 termsrv.dll If I knew how to create it, I would help you too

**Comment by Sharkta2** [comment]
[10.0.22621.3374]
SingleUserPatch.x64=1
SingleUserOffset.x64=C3F3
SingleUserCode.x64=Zero
DefPolicyPatch.x64=1
DefPolicyOffset.x64=1C2E5
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=9D661
LocalOnlyCode.x64=jmpshort
SLInitHook.x64=1
SLInitOffset.x64=29ED0
SLInitFunc.x64=New_CSLQuery_Initialize

[10.0.22621.3374-SLInit]
bServerSku.x64=12BF50
bRemoteConnAllowed.x64=12BF64
bFUSEnabled.x64=12BF74
bAppServerAllowed.x64=12BF5C
bMultimonAllowed.x64=12BF68
lMaxUserSessions.x64=12BF54
ulMaxDebugSessions.x64=12BF70
bInitialized.x64=12BF4C

**Comment by loyejaotdiqr47123** [comment]
[Add support : 22621.3374](https://github.com/loyejaotdiqr47123/rdpwrap/commit/a340d3412007a3cc8c5e69d254968d5b3d838e50)

**Comment by loyejaotdiqr47123** [comment]
[add support : 22621.3374](https://github.com/loyejaotdiqr47123/rdpwrap/commit/88c1f8c997480d6473cb14eac351a7bb3bf520f2)

**Comment by lucky1luc** [comment]
@Sharkta2 it works. Thanks!

**Comment by TanMan1217** [comment]
@Sharkta2 thanks very much! I can confirm it works!

**Comment by happysea0107** [comment]
> Try https://github.com/llccd/RDPWrapOffsetFinder/releases
> 
> Execute EXE and add values to your `rdpwrap.ini` :)
> 
> Video: https://t.me/rdpwrap/69189

I can't open the video, could you please briefly describe how to operate it?

**Comment by sashaqwert** [comment]
@happysea0107 

<details>
<summary>CLICK ME</summary>

https://github.com/stascorp/rdpwrap/assets/24974508/09444127-a78d-4b8b-9137-561ff81a939d

</details>

**Comment by sashaqwert** [comment]
Solution here: https://github.com/stascorp/rdpwrap/issues/2528#issuecomment-2022318278 (above)

**Comment by cifarmamarco** [comment]
> Windows 11 10.0.22621.3374 
Yes, it's not supported, I'd say I'm in a terrible situation because it doesn't even work with update, and the dll in circulation is 3374. Does anyone have a solution please?

**Comment by cifarmamarco** [comment]
> [termsrv.zip](https://github.com/stascorp/rdpwrap/files/14766828/termsrv.zip)

no job with  this  dll

**Comment by happysea0107** [comment]
> @happysea0107
> 
> CLICK ME

Thanks! I am still unable to open the video properly, but after editing the existing rdpwrap.ini file with a text editor, I have now resolved the issue.

**Comment by Lyrxcx** [comment]
@happysea0107 would u mind posting your ini im still having the problem i decided to upgrade from windows 11 home to pro thinking it would resolve the problem but it still says its not supported

**Comment by lucky1luc** [comment]
> @happysea0107 would u mind posting your ini im still having the problem i decided to upgrade from windows 11 home to pro thinking it would resolve the problem but it still says its not supported

if you have pro you do not need rdpwrapper, because in pro it is enabled

**Comment by Lyrxcx** [comment]
i keep getting a access denied now

**Comment by lucky1luc** [comment]
> i keep getting a access denied now

check the firewall on your new pro machine. the port for rdp must be opened on your windows firewall

**Comment by Lyrxcx** [comment]
i turned off my firewall completely

**Comment by Lyrxcx** [comment]
the reason im trying to use rdp is to run my computer 2x at once im my computer and vm

**Comment by Lyrxcx** [comment]
If anyone is still having the problem here is the fix https://github.com/BobbyCephy/rdpwrap.ini/blob/master/rdpwrap.ini

**Comment by dn2182** [comment]
https://github.com/stascorp/rdpwrap/issues/2535#issuecomment-2027969145

Here is how to fix...

**Comment by dn2182** [comment]
Hope this helps.... 
You need to add offsets in the right order, here is my .ini as an example....
[rdpwrap.ini.zip](https://github.com/stascorp/rdpwrap/files/14832365/rdpwrap.ini.zip)

**Comment by DF87SE** [comment]
O meu está ok, mas só permiti uma conexão por usuário, não permite multiplos, alguém pode me ajudar com isso
